docs/index.rst

```diff
@@ -261,17 +261,41 @@ would be:
 
 To use this post processor you have to tell ``render_block`` where it's
 located. If above code sample lives in the Python module
-``myapp.sekizai_processors`` you could use it like this::
+``myapp.sekizai_processors`` you could use it like this
 
-    ...
-    {% render_block "js" postprocessor "myapp.sekizai_processors.spaceless_post_processor" %}
-    ...
+.. code-block:: html
 
+   ... 
+   {% render_block "js" postprocessor "myapp.sekizai_processors.spaceless_post_processor" %}
+   ...
 
-It's also possible to pre-process data in ``{% addtoblock %}`` like this::
+It's also possible to pre-process data in ``{% addtoblock %}`` like this
 
-    {% addtoblock "css" preprocessor "myapp.sekizai_processors.processor" %}
+.. code-block:: html
 
+   {% addtoblock "css" preprocessor "myapp.sekizai_processors.processor" %}
+
+Control Pre- and Postprocessors
+-------------------------------
+
+To avoid the repetitive tasks of writing ``postprocessor "..."`` and/or ``preprocessor "..."`` in
+your templatetags, you can define them globally. To your ``settings.py`` add
+
+.. code-block:: python
+
+    SEKIZAI_PROCESSORS = {
+        'pre': {
```

Comment:
Not sure about this structure. How about ``SEKIZAI_DEFAULT_PREPROCESSORS``/``SEKIZAI_DEFAULT_POSTPROCESSORS`` both being dicts with namespaces as keys, import path as values?